src/core/_buttons.css

```diff
@@ -19,6 +19,11 @@ input[type="button"] {
   border-radius: var(--border-radius);
   border: var(--button-border);
   cursor: pointer;
+  -webkit-transition: border 0.2s;
```

Comment:
Gulp is also a good way to build tghe CSS. There are many repos that use Gulp and there are also many packages to extend this Process.